code/game/gamemodes/miniantags/guardian/types/healer.dm

```diff
@@ -51,7 +51,7 @@
 		if(iscarbon(target))
 			changeNext_move(CLICK_CD_MELEE)
 			if(heal_cooldown <= world.time && !stat)
-				var/mob/living/carbon/C = target
+				var/mob/living/carbon/human/C = target
```

Comment:
why typecast as human under and `iscarbon` check?

Reply:
`adjustBruteLoss` with the `robotic` arg only applies to humans. The reason for the typecast to human is I cannot see any scenario where a healing holopara would have to heal a carbon that isnt a human.